tests/Tests/500__jupyterhub/test.robot

```diff
@@ -10,67 +10,70 @@ Suite Teardown   End Web Test
 
 Can Spawn Notebook
-   [Tags]  Sanity  Smoke
-   Fix Spawner Status
-   Select Notebook Image  s2i-generic-data-science-notebook
-   Select Notebook Image  s2i-minimal-notebook
-   Select Container Size  Small
-   # Cannot set number of required GPUs on clusters without GPUs anymore
-   #Set Number of required GPUs  9
-   #Set Number of required GPUs  0
-   Add Spawner Environment Variable  env_one  one
-   Remove Spawner Environment Variable  env_one
-   Add Spawner Environment Variable  env_two  two
-   Remove Spawner Environment Variable  env_two
-   Add Spawner Environment Variable  env_three  three
-   Remove Spawner Environment Variable  env_three
-   Add Spawner Environment Variable  env_four  four
-   Add Spawner Environment Variable  env_five  five
-   Add Spawner Environment Variable  env_six  six
-   Remove Spawner Environment Variable  env_four
-   Remove Spawner Environment Variable  env_five
-   Remove Spawner Environment Variable  env_six
-   Spawn Notebook  same_tab=${False}
-   Login To Openshift  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
-   ${authorization_required} =  Is Service Account Authorization Required
-   Run Keyword If  ${authorization_required}  Authorize jupyterhub service account
-   Wait Until Page Contains Element  xpath://div[@id="jp-top-panel"]  timeout=60s
-   Sleep  3
-   Maybe Close Popup
-   ${is_launcher_selected} =  Run Keyword And Return Status  JupyterLab Launcher Tab Is Selected
-   Run Keyword If  not ${is_launcher_selected}  Open JupyterLab Launcher
-   Launch a new JupyterLab Document
-   Close Other JupyterLab Tabs
-
-Can Launch Python3
-   [Tags]  Sanity  TBC
-   Launch Python3 JupyterHub
+    [Tags]  Sanity  Smoke
+    ...     ODS-1808
+    Fix Spawner Status
+    Select Notebook Image  s2i-generic-data-science-notebook
+    Select Notebook Image  s2i-minimal-notebook
+    Select Container Size  Small
+    # Cannot set number of required GPUs on clusters without GPUs anymore
+    #Set Number of required GPUs  9
+    #Set Number of required GPUs  0
+    Add Spawner Environment Variable  env_one  one
+    Remove Spawner Environment Variable  env_one
+    Add Spawner Environment Variable  env_two  two
+    Remove Spawner Environment Variable  env_two
+    Add Spawner Environment Variable  env_three  three
+    Remove Spawner Environment Variable  env_three
+    Add Spawner Environment Variable  env_four  four
+    Add Spawner Environment Variable  env_five  five
+    Add Spawner Environment Variable  env_six  six
+    Remove Spawner Environment Variable  env_four
+    Remove Spawner Environment Variable  env_five
+    Remove Spawner Environment Variable  env_six
+    ${version-check}=   Is RHODS Version Greater Or Equal Than  1.18.0
+    IF  ${version-check}==True
+        FOR    ${env_var}    IN    @{UNSUPPORTED_VAR_NAMES}
+            Verify Unsupported Environment Variable Is Not Allowed    ${env_var}
+        END
```

Comment:
Will it be useful to have a `Verify Supported Environment Variable Is Allowed` and check a list of correct ones?